### DISCUSSION TITLE
Url of connection to multiple schemas for postgresql

### DISCUSSION THREAD

**Discussion by mcra02** [prisma/prisma#2223] [Q&A] [answered]
A good idea would be to add support for multiple schemas in the connection url to postgresql. Example "schema = public, myschema1, mysqchema2" or "search_path = public, myschema1, myschema2".
Prisma is great!

**Comment by ryands17** [comment, marked as answer]
Hey @mcra02 :wave: 
Can you pls create a Feature Request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and we can look into this from there :smile:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
